README.md

```diff
@@ -1,80 +1,201 @@
+images collected during the collimation of a telescope. A "pupil plate"
+is a defocused star that is defocused so much that it appears as a donut.
+The shape of that donut can be used to characterize
+(and hopefully be able to eliminate) optical distortion.
 
-### Purpose
-The purpose of this tool is to facilitate processing of pupil plate
-images collected during the collimation of a Telescope.
 The goal of this software tool is to help the user isolate which
 pupil plate image is the best out of a pre-exposed set of images by
-identifying a set of quantitative characteristics processed through
-photometric analysis tools.
-
-At the end of this development the team should have a tool that they
-can use to process many fits file images, quantify the best image to
-move forward with, and visualize how the various tip/tilts of the M2
-mechanism affect various photometric quantities.
-This software should be readily available at 1m sites and should
-process images quickly enough to not burden the team 
-(100 images in 5 minutes).
-
-### Problem
-During collimation engineers use a script to take a number of out
+identifying the best position for the tip and tilt of the M2 mechanism.
+
+## Installation
+The code is meant to be run in a docker file. You can pull the docker image by typing 
+the following in the terminal: 
+```
+docker pull docker.lco.global/pupe-pat:0.1.0
+```
+You can also install the code directly 
+(after installing the dependencies, hopefully in an environment)
+by running
+```
+git clone https://github.com/lcogt/pupe-pat
+cd pupe-pat
+python setup.py install
+```
+
+### Prerequisites
+Docker needs to be installed on your system and you need to be a docker user.
+`eng` is probably the best option. `optics-support` is probably the best 
+machine to run this code on at the moment. 
+
+### Dependencies
+- sphinx
+
+- numpy
+
+- astropy
+
+- lcogt_logging
+
+- sep
+
+- matplotlib
+
+- watchdog
+
+- scipy
+
+- emcee (only needed for manual analysis)
+
+- corner (only needed if emcee is installed)
+
+- pypdf2
+
+
+[//]: # (Describe how to run tests in the project)
+## Tests
+Currently the tests are run manually. Our test data set was taken on
+kb05 at bpl on 20180409. We use the whole raw directory of images.
+To reduce the test data run the following on the `optics-support` machine
+as the `eng` user:
+```
+docker pull docker.lco.global/pupe-pat:{0.1.0}
+docker run --rm -it -v /home/eng/pupe-pat-test-data:/input -v /home/eng/pupe-pat-tests-{0.1.0}:/output docker.lco.global/pupe-pat:{0.1.0} run_pupepat --output-dir /output --input-dir /input
+```
+Replace `{0.1.0}` with the current version.
+The tests pass if all of the blue circles should align with the donuts
+in the PDF files produced in the output directory.
+
+Eventually this procedure will be replaced with automatic testing in Jenkins.
+
+[//]: # (Details on how to deploy the project; if the project is deployed using
+CD, just put a link to the job in this section)
+## Deployment
```

Comment:
It may not be obvious to user what the difference is between installation and deployment. You could maybe say something at the beginning of the README about which sections are relevant to users and which to developers

Reply:
Done.